securedrop_client/gui/widgets.py

```diff
@@ -679,6 +679,7 @@ def setup(self, controller):
         """
         self.controller = controller
         self.source_list.setup(controller)
+        self.source_list.delete_source_by_uuid.connect(self.delete_source)
```

Comment:
@redshiftzero here's where the signal for deleting the source by UUID is connected to the event handler.